config/app_config.yml

```diff
@@ -3,7 +3,7 @@ default:
 buckinghamshire:
   title: Manage service listing
   directories:
-    - name: FIS
-      value: Family Information Service
-    - name: BOD
-      value: Buckinghamshire Online Directory
\ No newline at end of file
+    - value: Family Information Service
```

Comment:
I have a feeling this will break some filters elsewhere. We don't have tests for them though so we'll need to check that manually (or write some tests... 😈)

Reply:
is this because `@counts["Family Information Service"]["archived"]` could be a bad idea?

Reply:
Oh no sorry, I read this wrong. Thought we'd renamed the "value" key and the user filters would break if we did, but I see now that we haven't. Ignore me!